src/connector/src/parser/unified/debezium.rs

```diff
@@ -226,20 +233,41 @@ pub fn parse_schema_change(
                     // handle default value expression, currently we only support constant expression
                     let column_desc = match col.access_object_field("defaultValueExpression") {
                         Some(default_val_expr_str) if !default_val_expr_str.is_jsonb_null() => {
-                            let mut value_text = default_val_expr_str.as_string().unwrap();
-                            // mysql timestamp is mapped to timestamptz, we use UTC timezone to
-                            // interpret its value
-                            if data_type == DataType::Timestamptz
-                                && matches!(*connector_props, ConnectorProperties::MysqlCdc(_))
-                            {
-                                value_text = timestamp_val_to_timestamptz(value_text.as_str()).map_err(|err| {
-                                    tracing::error!(target: "auto_schema_change", error=%err.as_report(), "failed to convert timestamp value to timestamptz");
-                                    AccessError::TypeError {
-                                        expected: "timestamp in YYYY-MM-DD HH:MM:SS".into(),
-                                        got: data_type.to_string(),
-                                        value: value_text,
-                                    }})?;
+                            let value_text: String;
+                            match *connector_props {
+                                ConnectorProperties::PostgresCdc(_) => {
+                                    // default value of non-number data type will be stored as
+                                    // "'value'::type"
```

Comment:
Does this always hold per some documentation or specification? Shall we avoid panicking here for robustness?

Reply:
If should always hold, since it comes from the `column_default` in [information_schema](https://www.postgresql.org/docs/15/infoschema-columns.html).
But we can also set the snapshot value to null if parse failed.